### PR TITLE
fix error message

### PR DESCRIPTION
**Pull request by johri21**

### CODE REVIEW

**Review by kelindar — APPROVED** [review]

File: module.go [hunk 1]
```diff
@@ -94,12 +95,12 @@ func (g *fngen) generate() lua.LGFunction {
 		switch len(out) {
 		case 1:
 			if err := out[0]; !err.IsNil() {
-				state.RaiseError(err.String())
+				state.RaiseError(err.Interface().(error).Error())
```
Comment (kelindar) [inline module.go:98]: We may need to check if `Interface()` is actually of `error` type.

**Review by kelindar — CHANGES_REQUESTED** [review]

File: module.go (hunk 1 above)
Comment (johri21) [reply module.go:98]: Good point Roman, I thought about it and removed it later
`&& err.Type().Implements(errorInterface)` the point being we do validation when we register a function, don't you think it's like doing it twice 🤔

Comment (johri21) [reply module.go:98]: the validation guarantees that the return type is error, no? 🤔

Comment (johri21) [reply module.go:98]: doing it [here](https://github.com/kelindar/lua/blob/master/module.go#L155)

**Review by kelindar — APPROVED** [review]

**Pull request merged**